client/client-portals/utils/icons/last.tsx

```diff
@@ -0,0 +1,11 @@
+const Last = () => (
+    <svg stroke="currentColor"
+      fill="currentColor"
+      strokeWidth="0"
+      viewBox="0 0 1024 1024"
+      height="1em"
+      width="1em"
```

Comment:
replace with `className="h-full w-full"`

Reply:
I have addressed this.